.github/workflows/python-tests.yml

```diff
@@ -15,25 +15,44 @@ jobs:
       matrix:
         python-version:
         - 3.8
+        - 2.7
+        - pypy3
         - 3.7
         - 3.6
         - 3.5
-        - 2.7
+        - pypy2
         os:
-        - ubuntu-18.04
+        - ubuntu-latest
         - ubuntu-16.04
         - macOS-latest
         # - windows-2019
         # - windows-2016
         env:
         - TOXENV: python
 
+    env:
+      TOX_PARALLEL_NO_SPINNER: 1
+
     steps:
     - uses: actions/checkout@master
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v1.1.1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Log Python version
+      run: >-
+        python --version
+    - name: Log Python location
+      run: >-
+        which python
+    - name: Cache pip
+      uses: actions/cache@v1
       with:
-        version: ${{ matrix.python-version }}
```

Comment:
@jaraco a bit of explanation needed here. This was the initial name of the arg at the time of my first PR but then they changed the name and deprecated it gradually, eventually, it stopped having any effect on the action. So the previous PR was accepted with this bug and basically used whatever was system's default `python` and not what `actions/setup-python` set.